src/components/dialog-full-screen/dialog-full-screen.pw.tsx

```diff
@@ -492,6 +493,20 @@ test.describe("Accessibility for DialogFullScreen", () => {
     await openButton.click();
     await checkAccessibility(page);
   });
+
+  test("should check accessibility with complex example", async ({
+    mount,
+    page,
+  }) => {
+    await mount(<WithComplexExample />);
+
+    const openButton = page
+      .getByRole("button")
+      .filter({ hasText: "Open DialogFullScreen" });
+    await openButton.click();
+    await page.waitForTimeout(300);
```

Comment:
**issue**: I wouldn't suggest using the fixed timeout here, as it could introduce flakiness and [is discouraged by Playwright themselves](https://playwright.dev/docs/api/class-frame#frame-wait-for-timeout).

Instead I would assert if the dialog is definitely open first before doing `checkAccessibility`. One way you could do this is to check if the title is present. Playwright will [automatically wait for the `toHaveText` assertion](https://playwright.dev/docs/test-assertions#auto-retrying-assertions) to pass:

```suggestion
    await expect(getDataElementByValue(page, "title")).toHaveText("Dialog Title");
```